meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/files/L4TConfiguration-RootfsRedundancyLevelABEnable.dtsi

```diff
@@ -0,0 +1,41 @@
+/dts-v1/;
```

Comment:
Why do we need our own copies of this file and logic after https://github.com/OE4T/meta-tegra/commit/d421bf5cfe1c3ad876984d12a4a6c7158d67d6b7 ?  Is this really needed?

Reply:
You are right, I think both of these can be removed now that we have removed the local custom flash xml files, will test

Reply:
Relates to https://github.com/OE4T/meta-mender-community/pull/19#issuecomment-2233429354